### ISSUE TITLE
Crashes using floor/ceiling texture mode on Windows 10

### ISSUE THREAD

**Issue by aaronfreed** [treellama/weland#15]
There are at least two, which may or may not be related:

* If another app is using the shapes file, it almost always crashes instantly.
* When applying textures, it almost always crashes within about ten seconds. This seems to happen particularly often when I’m applying landscape textures.

The textures on the polygons also only occasionally display correctly. I haven’t yet figured out what causes this.

tl;dr: I don’t know why I still use Windows.

**Comment by treellama** [comment]
Does this happen with no other apps using the shapes file?

**Comment by aaronfreed** [comment]
Regarding the second one, I’m… not completely sure. I haven’t gotten it to crash yet in my most recent test, but that doesn’t mean it won’t five minutes after I write this. :doog:

This may mean I need to amend my initial report to “it will _probably_ crash if another app is using the shapes file, and if you try to apply textures when another app is using the shapes, it will almost certainly crash within about ten seconds.” I’m not really sure, though.
